lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -446,6 +452,20 @@ def _prepare_environment(  # noqa: C901
                                 min_cap.nic_count, int
                             ), f"actual: {min_cap.nic_count}"
                             node_runbook.nic_count = min_cap.nic_count
+                            assert isinstance(
+                                min_cap.data_disk_count, int
+                            ), f"actual: {min_cap.data_disk_count}"
+                            node_runbook.data_disk_count = min_cap.data_disk_count
+                            assert isinstance(
+                                min_cap.data_disk_caching, str
```

Comment:
This could be tested as one of the three allowed values

Reply:
It already covered in schema.py